core/src/display_object/edit_text.rs

```diff
@@ -803,6 +803,9 @@ impl<'gc> EditText<'gc> {
             if !is_word_wrap {
                 edit_text.bounds.set_x(new_x);
                 edit_text.bounds.set_width(width);
+            } else {
+                let max_width = edit_text.static_data.text.bounds.x_max;
+                edit_text.bounds.set_width(max_width);
```

Comment:
We could move the `is_word_wrap` check a little higher to avoid unnecessarily calculating `new_x` in the word wrap case:

```
if !is_word_wrap {
  let width = intrinsic_bounds.width() + padding;
  let new_x = match autosize { // ...etc }
  edit_text.bounds.set_x(new_x);
  edit_text.bounds.set_width(width);
} else {
  // ... etc.
}
  let height = intrinsic_bounds.height() + padding;
  edit_text.bounds.set_height(height);
```

Also I don't think `edit_text.bounds.set_width(max_width)`  is necessary -- if wordWrap+autosize are on, the width of the text field shouldn't change. Example SWF:

[autosize1.zip](https://github.com/ruffle-rs/ruffle/files/9397764/autosize1.zip)
```
txt._width = 200;
txt.autoSize = "left";
trace(txt._width);
```
Traces 200 in FP, but 98 in this PR because the text field ends up resetting to the SWF dimensions.